src/zkevm_specs/evm/execution/addmod.py

```diff
@@ -0,0 +1,65 @@
+from ..instruction import Instruction, Transition
+from ..opcode import Opcode
+from zkevm_specs.util import FQ, RLC
+
+# Returns 1 when a is lower than b, 0 otherwise
+def lt_u256(instruction: Instruction, a: RLC, b: RLC) -> FQ:
```

Comment:
It might be the case that this fn can be re-used for other stuff. Maybe not now but let's keep an eye to `MULMOD` and others since we might need to shar the fn across them.

Reply:
I reused this function for `MULMOD`, maybe it would be good to move it to `instruction.py`.